build/metadata/github-actions/action.yml

```diff
@@ -24,6 +24,9 @@ inputs:
     default: "false"
     description: "Enable verbose debug logging."
     required: false
+  dir:
+    description: "Path to existing checkout of the repository. You may use this if your action checkouts multiple repositories. The default value is `$GITHUB_WORKSPACE`."
+    required: false
```

Comment:
we don't set an explicit default sting here, because it reads the default from environment variable `$GITHUB_WORKSPACE`